packages/remix-server-runtime/index.ts

```diff
@@ -74,21 +74,3 @@ export type {
   UploadHandler,
   UploadHandlerPart,
 } from "./reexport";
-
-export type {
-  AgnosticDataRouteObject,
-  AgnosticIndexRouteObject,
-  AgnosticNonIndexRouteObject,
-  AgnosticRouteObject,
-  InitialEntry,
-  Location,
-  MemoryHistory,
-  StaticHandler,
-} from "./router";
-export {
-  createMemoryHistory,
-  matchRoutes,
-  unstable_createStaticHandler,
-} from "./router";
-export type { Update } from "./router/history";
-export type { AgnosticRouteMatch } from "./router/utils";
```

Comment:
No longer have this dup code, can import these right from `@remix-run/router` over in `remix-testing` instead of exporting through here